### ISSUE TITLE
Support for SSL/TLS options?

### ISSUE THREAD

**Issue by brentstrange** [ForbesLindesay/http-basic#6]
Hey Forbes,
Is it possible that you could add the options/support for SSL/TLS?

Basically at:
https://github.com/ForbesLindesay/http-basic/blob/master/index.js#L225	    

Add the following options:

rejectUnauthorized: options.rejectUnauthorized,
cert : options.cert,
pfx : options.pfx,
passphrase : options.passphrase,
key : options.key,
ca : options.ca

...and have those options bubble up into the 'sync-request' library?

**Comment by ForbesLindesay** [comment]
Yes, just submit a pull request, ensuring that they maintain their current default values.
